platform-metrics/src/main/java/org/hypertrace/core/serviceframework/metrics/PlatformMetricsRegistry.java

```diff
@@ -384,6 +385,10 @@ public static <T extends Number> T registerGauge(
     return number;
   }
 
+  public static MultiGauge registerMultiGauge(final String gaugeName) {
```

Comment:
Do we want to just expose a micrometer gauge like this? Seems bad from an abstraction point of view (as you can see you had to expose the tags util, too)

Reply:
I see we did that with other meters, so maybe that ship has sailed... I'll leave it up to you if you want to add a simple abstraction to this one or not.

Reply:
I can understand the issue. But, not sure about the solution you're proposing.

Reply:
You'd likely need to create an abstract representation of a multigauge that internally routes calls to the micrometer instance. So the main api it would likely need to expose is one that takes in a list of measurements, where each measurement pairs a set of labels (Strings) and a value.